pages/tools/slack/notifications-general-communication.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Slack communication and notification best practices
+questions:
+  - admins-slack
+redirect_from:
+  - /tools/slack/rules/
```

Comment:
```suggestion
```

These two lines are the source of the build problem. There is already another page (`tools/slack/guidelines.md`) that redirects from this location, and that has to be a 1:1 relationship. Since this is a brand new page, it doesn't need the `redirect_from` at all – that's mostly for when a page is renamed, so old links don't break.